src/core/prism.ts

```diff
@@ -162,7 +162,13 @@ export class Prism {
 	 * Usually a language definition like `Prism.languages.markup`.
 	 * @returns The highlighted HTML.
 	 * @example
-	 * Prism.highlight('var foo = true;', 'javascript');
+	 * import { Prism } from './src/core/prism';
+	 * import javascript from './src/languages/prism-javascript';
+	 * const prism = new Prism();
+	 * prism.components.add(javascript);
+	 * prism.highlight('var foo = true;', 'javascript')
+	 * // Returns:
+	 * `<span class="token keyword">var</span> foo <span class="token operator">=</span> <span class="token boolean">true</span><span class="token punctuation">;</span>`
```

Comment:
I think
```
* ```ts
* import { Prism } from './src/core/prism';
* import javascript from './src/languages/prism-javascript';
* const prism = new Prism();
* prism.components.add(javascript);
* prism.highlight('var foo = true;', 'javascript')
* ```
* Returns:
* ```html
* <span class="token keyword">var</span> foo <span class="token operator">=</span> <span class="token boolean">true</span><span class="token punctuation">;</span>
* ```
```
produces a slightly better result
<img width="500" alt="image" src="https://github.com/user-attachments/assets/d8ed3bd2-fca5-4270-b273-f50c4f81f70e" />


What do you think?